util/fipstools/acvp/modulewrapper/modulewrapper.cc

```diff
@@ -12,6 +12,7 @@
  * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
  * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE. */
 
+#include <algorithm>
```

Comment:
what does this bring in?

Reply:
`std::min`, which I used in the MCT for SHAKE